bitnami/redis/Chart.yaml

```
@@ -25,4 +25,4 @@ name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
   - http://redis.io/
-version: 12.8.3
+version: 12.8.4
```

As it is a new feature addition instead of bug fixing, it's better to bump the minor version
```suggestion
version: 12.9.0
```